en/theme/material/assets/css/theme.css

```
@@ -0,0 +1,929 @@
+    --md-primary-fg-shade-color: #eb4f63;
+    --md-accent-fg-color: #e46700;
+    --md-redoc-example-bg-color: #2b3339;
+    /* END OF - Default Material Theme Overrides  */
+    
+    /* START OF - Icon Colors */
+    --icon-bg-color-opacity: 1;
+    --icon-color-1-bg: rgba(233, 246, 252, var(--icon-bg-color-opacity));
+    --icon-color-1-fg: #27aae1;
+    --icon-color-2-bg: rgba(255, 236, 246, var(--icon-bg-color-opacity));
+    --icon-color-2-fg: #cf3b89;
+    --icon-color-3-bg: rgb(231, 246, 245, var(--icon-bg-color-opacity));
```

Stylelint (17.2.0)

[error] Expected modern color-function notation (color-function-notation)

(color-function-notation)